packages/salesforcedx-vscode-aura/package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "salesforcedx-vscode-aura",
+  "displayName": "Lightning",
+  "description": "Provides code-editing features for Lightning",
+  "qna": "https://github.com/forcedotcom/salesforcedx-vscode/issues",
+  "bugs": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode/issues"
+  },
+  "repository": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode"
+  },
+  "icon": "images/VSCodeLWC.png",
+  "galleryBanner": {
+    "color": "#ECECEC",
+    "theme": "light"
+  },
+  "version": "45.6.0",
+  "publisher": "salesforce",
+  "preview": true,
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.26.0"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "dependencies": {
+    "@salesforce/salesforcedx-utils-vscode": "45.6.0",
+    "chalk": "^2.4.2",
+    "change-case": "^3.1.0",
+    "lightning-lsp-common": "1.0.4",
+    "lwc-language-server": "1.8.3",
+    "opn": "^5.4.0",
+    "vscode": "1.1.17",
+    "vscode-extension-telemetry": "0.0.17",
+    "vscode-languageclient": "^3.5.1"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.0.0",
+    "@types/mocha": "2.2.38",
+    "@types/opn": "^5.1.0",
+    "@types/node": "8.9.3",
+    "@types/sinon": "^2.3.2",
+    "typescript": "3.1.6",
+    "chai": "^4.0.2",
+    "cross-env": "^5.0.5",
+    "mocha": "3.2.0",
+    "sinon": "^2.3.6"
+  },
+  "scripts": {
+    "vscode:prepublish": "npm prune --production",
+    "vscode:package": "vsce package",
+    "compile": "tsc -p ./",
+    "lint": "tslint --project .",
+    "watch": "tsc -watch -p .",
+    "clean": "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
+    "postinstall": "node node_modules/.bin/vscode-install",
```

Comment:
This target fails when running it on windows (after running `npm install`) but it doesn't on a mac.

```
lerna ERR! npm run postinstall [salesforcedx-vscode-aura] Output from stderr:
npm WARN lifecycle The node binary used for scripts is C:\Users\lcamp\Documents\github\salesforcedx-vscode\node_modules\node\bin\node.exe but npm is using C:\Program Files\nodejs\node.exe itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.
C:\Users\lcamp\Documents\github\salesforcedx-vscode\packages\salesforcedx-vscode-aura\node_modules\.bin\vscode-install:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^

SyntaxError: missing ) after argument list
    at new Script (vm.js:74:7)
    at createScript (vm.js:246:10)
    at Object.runInThisContext (vm.js:298:10)
    at Module._compile (internal/modules/cjs/loader.js:670:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:744:10)
    at startup (internal/bootstrap/node.js:238:19)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! salesforcedx-vscode-aura@45.6.0 postinstall: `node node_modules/.bin/vscode-install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the salesforcedx-vscode-aura@45.6.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```